core/src/components/input/input.ionic.scss

```diff
@@ -1,26 +1,127 @@
 @use "../../themes/ionic/ionic.globals.scss" as globals;
 @use "./input.common";
-@forward "./input.ionic.outline.scss";
-
+@forward "./input.ionic.solid.scss";
 // Ionic Input
 // --------------------------------------------------
 
 :host {
   --color: #{globals.$ion-primitives-neutral-1200};
   --border-width: #{globals.$ion-border-size-025};
   --border-color: #{globals.$ion-primitives-neutral-500};
+  --border-radius: #{globals.$ion-border-radius-100};
```

Comment:
Similarly, can we move all styles specific to `shape` to the shape class such as `input-fill-round` so that if we change the default we don't have to move styles?